guides/support-and-on-call.md

```diff
@@ -0,0 +1,156 @@
+
+With our default support plan we will manage all code, maintenance, updates and
+uptime monitoring and provide security event monitoring intended to detect and
+prevent attacks. Users can also use our helpdesk service for help with any
+aspect of their site.
+
+Our Managed Plus plan is for clients with higher assurance requirements. It
+includes all the fully-managed features of our default support plan, plus a
+quarterly site review meeting, a named delivery manager and an enhanced SLA
+with faster response targets.
+
+We provide support for clients and users of these services in three ways:
+
+* Clients and users of these services can raise a **ticket** with us about
+  usability issues, errors or faults. Sometimes what they’re asking for is a
+  larger feature request, which needs separate handling.
+* We do proactive **maintenance** for our client’s sites and services, for
+  example by applying security updates.
+* We have monitoring systems for our services that continually check whether
+  they are operating correctly, and whether there are any warning signs of
+  problems. When problems are detected they raise an **alert** for the team to
+  investigate and resolve.
+
+That support is handled by the team in two ways:
+
+* We provide in-hours (10.00–18.00) support for all our clients, resolving
+  tickets and alerts and doing maintenance. We call this **first line
+  support**, and it’s staffed by a developer and an ops engineer on rota.
+* We provide out-of-hours (24/7) support for some of our clients, responding
+  only to critical alerts and emergency tickets. We call this **on call**, and
+  it’s staffed by a developer or an ops engineer through an opt-in rota.
+
+## Being on first line support
+
+Support tickets come in from clients and users every day. These tickets could
+relate to any of the products we currently support, which are a mixture of
+rails apps and Wordpress implementations, spread across clients all over the
+private sector, including central government.
+
+Tickets could be about any aspect of the service, from a production outage to a
+typo, or something that turns out to be a request for more development work
+from a client.
+
+In order to make sure all tickets are effectively dealt with, a dedicated
+person will work on first line support for a week at a time, inside office
+hours.
+
+There are lots of details about [our approach to first line support in the
+playbook](http://playbook.dxw.com/#/?id=tickets).
+
+### Which things are my responsibility when I am on first line support?
+
+While on support, your focus is on monitoring incoming alerts and tickets,
+responding to them or delegating to colleagues as appropriate.
+
+#### Alerts
+
+We use monitoring and alerts to notify you when an issue occurs, and provide
+[run-books] to help you respond to events and restore service for our clients.
+
+[run-books]: https://git.govpress.com/ops/docs
```

Comment:
Should this be an actual link?

Reply:
If you mean in terms of markdown syntax, this provides a link for the reference (`[run-books]`) in the paragraph above (and also another paragraph further down the page).

If you mean in terms of that URL doesn't resolve to anything for you, then we'll need to make sure you have the right access permissions for it.

Reply:
Ah my mistake, I misunderstood the syntax!